.github/workflows/test.yml

```diff
@@ -38,27 +38,31 @@ jobs:
         if: "github.ref != 'refs/heads/master'"
 
   go-tests:
-    name: Go Tests
-    runs-on: ubuntu-latest
+    name: Go Tests on ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-10.15]
+    runs-on: ${{ matrix.os }}
```

Comment:
move `runs-on` before `strategy`?

Reply:
Actually should prefer everything after `strategy/matrix` so that we define matrix before using it?